example/src/App.tsx

```diff
@@ -37,6 +38,7 @@ const App: React.FC<{}> = () => {
           </PTZProvider>
         </Route>
         <Route path={`/tabs`} exact={true} component={TabsPage} />
+        <Route path={`/datepicker`} exact={true} component={DatePickerComp} />
```

Comment:
DatePickerComp should be renamed to DatePickerPage

Reply:
Done